airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/models/airbyte_protocol.py

```diff
@@ -14,51 +14,51 @@
 
 
 class Type(Enum):
-    RECORD = "RECORD"
-    STATE = "STATE"
-    LOG = "LOG"
-    SPEC = "SPEC"
-    CONNECTION_STATUS = "CONNECTION_STATUS"
-    CATALOG = "CATALOG"
+    RECORD = 'RECORD'
```

Comment:
this looks like formatting churn. You probably want to back this out, so it doesn't conflict with upstream?